build.assets/docker/buildbox.dockerfile

```diff
@@ -13,9 +13,9 @@ COPY passwd /etc/passwd
 
 # Install build tools, dev tools and Go:
 RUN apt-get update && apt-get -t stretch-backports install -y libc6-dev libudev-dev && \
-	apt-get install -y curl make git gcc tar gzip vim screen dumb-init
+	apt-get install -y curl make git gcc tar gzip vim screen dumb-init bzr
```

Comment:
One of the dependencies(launchpad.net/gocheck) can be downloaded only with bazaar. Someone should rewrite code in the future to use v2 of that library.

Reply:
Why don't we do this in this PR?

Reply:
Because there is a lot of places with gocheck and I have to work on Helm migration. Usage of vendoring in source code will solve the problem for now, but as I said someone should return to the problem later.